templates/events/details.html

```diff
@@ -403,6 +418,54 @@ <h3>Meld deg av {{ event.title }} </h3>
                 </div>
             </div>
 
+            {% if event.event_type is 1 and user_status.status or user_attending %}
```

Comment:
Don't use `is` for comparing numbers in Python.
`is` tests if the objects compared are the same. It does work for small integers(up to 256), but this is bad practice as it only works because small integers are mapped to the same object in CPython.